src/__tests__/design-tokens.test.ts

```diff
@@ -2,12 +2,15 @@
 // SPDX-License-Identifier: Apache-2.0
 import { requireDesignTokensFile } from './utils';
 
+const themeNames = ['classic', 'visual-refresh'];
+
 describe('Design tokens artifacts', () => {
-  test.each<string>(['classic', 'visual-refresh'])(
-    `Design tokens JSON for %s matches the snapshot`,
-    (themeName: string) => {
-      const jsonContent = requireDesignTokensFile(`index-${themeName}.json`);
-      expect(jsonContent).toMatchSnapshot(themeName);
-    }
-  );
+  test.each<string>(themeNames)(`Design tokens JSON for %s matches the snapshot`, (themeName: string) => {
+    const jsonContent = requireDesignTokensFile(`index-${themeName}.json`);
+    expect(jsonContent).toMatchSnapshot(themeName);
+  });
+  test.each<string>(themeNames)(`Design tokens JSON schema is generated`, (themeName: string) => {
+    const jsonSchema = requireDesignTokensFile(`index-${themeName}-schema.json`);
+    expect(jsonSchema).not.toBeUndefined();
```

Comment:
Could we also take a snapshot here?

Reply:
I didn't do it because the format only depends on how many contexts we have in a theme, and that is tested in theming-core